docs/configuration.md

```diff
@@ -0,0 +1,261 @@
+# Connector Configuration Reference
+
```

Comment:
Eno, Do we have any existing documentation that describes the difference between "connector" and "connection?"

Reply:
No -- it's only a detail here. Other languages make this easier to understand.